gobblin-scheduler/src/main/java/gobblin/scheduler/JobScheduler.java

```diff
@@ -343,45 +350,50 @@ public void onFileCreate(File file) {
        */
       @Override
       public void onFileChange(File file) {
-        int pos = file.getName().lastIndexOf(".");
-        String fileExtension = pos >= 0 ? file.getName().substring(pos + 1) : "";
+        String fileExtension = Files.getFileExtension(file.getName());
+        if (fileExtension.equalsIgnoreCase(SchedulerUtils.JOB_PROPS_FILE_EXTENSION)) {
+          try {
```

Comment:
Can you put one log msg here, to indicate the detection of the ".properties" file change, like line 376 for ".job" change?